packages/cxl-lumo-styles/src/utils.js

```diff
@@ -14,9 +15,9 @@ export const registerGlobalStyles = (styles, options) => {
   }
 
   styles.forEach((cssResult) => {
-    if (!(cssResult instanceof CSSResult)) {
-      throw new Error('An item in styles is not of type CSSResult. Use `unsafeCSS` or `css`.');
-    }
+    // if (!(cssResult instanceof CSSResult)) {
+    //   throw new Error('An item in styles is not of type CSSResult. Use `unsafeCSS` or `css`.');
+    // }
```

Comment:
Can we just throw this code out?

Reply:
We control the input. I don't see much of a reason for this check.

Reply:
Can remove it, as code works as expected. But this check seems like no longer valid.